src/Generator/ObjectGenerator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Code\Generator;
+
+final class ObjectGenerator
```

Comment:
Discussed in private call: we will not implement this for now

Reply:
We'll need a new issue for this - will create one after clearing up ideas about this, for now, and for now, we will raise an exception in `::fromReflection()`, when an `object` as default value is encountered (something like "unsupported, for now")